### ISSUE TITLE
Universe CI warning from SignalR

### ISSUE THREAD

**Issue by JamesNK** [aspnet/SignalR#2473]
```
C:\projects\universe\modules\SignalR\src\Microsoft.AspNetCore.SignalR.Specification.Tests\Microsoft.AspNetCore.SignalR.Specification.Tests.csproj : warning KRB2003: Found a duplicate PackageReference for Internal.AspNetCore.Sdk. Restore results may be unpredictable. [C:\Users\appveyor\.dotnet\buildtools\korebuild\2.2.0-preview1-17089\KoreBuild.proj]
C:\projects\universe\modules\SignalR\src\Microsoft.AspNetCore.SignalR.Specification.Tests\Microsoft.AspNetCore.SignalR.Specification.Tests.csproj : warning KRB2003: Found a duplicate PackageReference for Internal.AspNetCore.Sdk. Restore results may be unpredictable. [C:\Users\appveyor\.dotnet\buildtools\korebuild\2.2.0-preview1-17089\KoreBuild.proj]
    2 Warning(s)
    0 Error(s)
```
https://ci.appveyor.com/project/aspnetci/universe/build/1.0.2341

**Comment by anurse** [comment]
@mikaelm12 I bet this is automatically imported. We manually import it into the Specifications.Tests.csproj. We should be able to just remove that import.

**Comment by mikaelm12** [comment]
Cool, I'll take care of this

**Comment by mikaelm12** [comment]
Should be fixed in: a4a8b49908c8d75669595dbeaae743ea77abfa9f
